### ISSUE TITLE
Ideas to improve zone config documentation

### ISSUE THREAD

**Issue by kuanluo** [cockroachdb/docs#1114]
I read https://www.cockroachlabs.com/docs/configure-replication-zones.html to prepare for designing how to show the zone config in the admin ui. From a non-technical perspective, I find the documentation a little hard to parse.

Not sure if it's entirely possible, just throwing some ideas here:
1. Add/call out the benefits of zone config in the first few paragraphs
2. Call out the importance of architecting the zone config right, and ways to avoid making bad decisions

@jseldess @sploiselle

**Comment by jseldess** [comment]
Thanks for sharing this feedback, @kuanluo. There's definitely room to improve this page, or possible split it into a few pages, totally agree. But Few follow-up questions:

1. In my opinion, the very first sentence summarizes what zone configs are all about: "In CockroachDB, you use replication zones to control the number and location of replicas for specific sets of data, both when replicas are first added and when they are rebalanced to maintain cluster equilibrium." In terms of "benefits" you're looking for, can you provide more details? Perhaps you mean adding more details about how replicas tie into fault-tolerance and how controlling location gives you flexibility to adapt based on use cases? Do the [scenario-based examples](https://www.cockroachlabs.com/docs/configure-replication-zones.html#scenario-based-examples) at the end help at all? 

2. This is a good point. We do have the [node/replica recommendations](https://www.cockroachlabs.com/docs/configure-replication-zones.html#nodereplica-recommendations) section, which points to some concrete best practices for zone configs, but there are probably more bad decisions we can help users avoid. If you have any additional insights, please share. Otherwise, @bdarnell, do you have suggestions here?

@sploiselle, also very interested in your overall sense of how to improve this content/make it more digestible.

**Comment by kuanluo** [comment]
Hi @jseldess,

> Perhaps you mean adding more details about how replicas tie into fault-tolerance and how controlling location gives you flexibility to adapt based on use cases? 

Yes exactly! 

> Do the scenario-based examples at the end help at all?

Yes, definitely. I wonder if surfacing/linking to them higher up on the page could be an idea to strengthen this page?

**Comment by jseldess** [comment]
These docs have been reworked a few times since this issue was opened.
